hooks/pre-push

```diff
@@ -1,21 +1,23 @@
 #!/usr/bin/env python3
-import sys, os
+import sys
 import re
-import string
 import subprocess
 
 args = sys.argv
 
 # get branch name
 bashCommand = 'git branch'
 process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
-output, error = process.communicate()
-branch_name = output[2:-1]
+output, _ = process.communicate()
+
+if '* master' in output.decode('utf-8').split('\n'):
```

Comment:
it can be solved more easily, check out `git branch --help`

Reply:
we've discussed this in private chat: `git branch --show-current` wasn't available at @sergey-byk0v version of git. Thus, not an issue